ProjectsNetwork/Startup.cs

```diff
@@ -31,6 +32,21 @@ public void ConfigureServices(IServiceCollection services)
                 .AddEntityFrameworkStores<ApplicationDbContext>();
             services.AddControllersWithViews();
             services.AddRazorPages();
+
+            services.AddAuthentication()
+                .AddMicrosoftAccount(microsoftOptions =>
+                 {
+                     microsoftOptions.ClientId = Configuration["Authentication:Microsoft:ClientId"];
+                     microsoftOptions.ClientSecret = Configuration["Authentication:Microsoft:ClientSecret"];
+                 })
+                  .AddGoogle(options =>
+                   {
+                       IConfigurationSection googleAuthNSection =
+                           Configuration.GetSection("Authentication:Google");
+
+                       options.ClientId = googleAuthNSection["ClientId"];
+                       options.ClientSecret = googleAuthNSection["ClientSecret"];
```

Comment:
You are not committing any file where "CLientID", "ClientSecret" are stored for both Google and Microsoft. This won't work since it won't find these values to use.